src/wmtk/utils/TriQualityUtils.hpp

```diff
@@ -10,6 +10,7 @@ namespace wmtk {
 struct NewtonMethodInfo
 {
     Eigen::MatrixXd neighbors; /// N x 4 matrix (2 vtx per triangle)
+    Eigen::VectorXi facet_ids; /// N x 1 vector of facet ids
```

Comment:
Sorry I didn't find where is facet_ids assigned

Reply:
It is assigned inside `AdaptiveTessellation::get_nminfo_for_vertex()` at this line:
https://github.com/wildmeshing/wildmeshing-toolkit/blob/51fbf54950b4bea6f930404ba63ab70e5d1dd14f/app/adaptive_tessellation/AdaptiveTessellation.cpp#L917